liquidity-pool-changes/src/agent.spec.js

```diff
@@ -0,0 +1,237 @@
+const BigNumber = require('bignumber.js');
+
+// pool mocking
+const mockToken0Address = '0xFAKETOKEN0ADDRESS'; // .token0()
+const mockToken1Address = '0xFAKETOKEN1ADDRESS'; // .token1()
+const mockPoolAddress = '0xFAKEPOOLADDRESS'; // .address
+const mockDecimals = 3;
+
+const mockPoolContract = {
+  // balanceOf gets mocked within each test case for modularity
+  token0: jest.fn().mockResolvedValue(mockToken0Address),
+  token1: jest.fn().mockResolvedValue(mockToken1Address),
+  decimals: jest.fn().mockResolvedValue(mockDecimals),
+  address: mockPoolAddress,
+};
+
+// combine the mocked provider and contracts into the ethers import mock
+jest.mock('forta-agent', () => ({
+  ...jest.requireActual('forta-agent'),
+  getEthersProvider: jest.fn(),
+  ethers: {
+    ...jest.requireActual('ethers'),
+    providers: {
+      JsonRpcBatchProvider: jest.fn(),
+    },
```

Comment:
The batch provider is not being used in this agent, so you don't actually need to mock it:
```suggestion
```